src/Compilers/VisualBasic/Test/Emit/ErrorHandling.vb

```diff
@@ -1037,6 +1037,51 @@ End]]>)
 ]]>)
         End Sub
 
+        <Fact>
+        Public Sub SynthesizingDefaultConstructorsWithMsCorLibMissing_Nothing()
+            Dim comp = CompilationUtils.CreateCompilation("
+Namespace System
+    Public Class Object
+    End Class
+End Namespace
+Class Test
+End Class
+")
+
+            CompilationUtils.AssertTheseDiagnostics(comp, <expected>
+BC30002: Type 'System.Void' is not defined.
+    Public Class Object
+    ~~~~~~~~~~~~~~~~~~~~
+BC30183: Keyword is not valid as an identifier.
```

Comment:
Yes. Since it is illegal for the user to define his own Object (because we produce BC30183), I want to make sure we always produce an error for that (blocking it). Do you think the scenario is remotely unlikely so that the test is not needed?

---
In reply to: [115349973](https://github.com/dotnet/roslyn/pull/18715#discussion_r115349973) [](ancestors = 115349973)